Add doc on how to uninstall ECK (#1435)
elastic/cloud-on-k8s: docs/uninstall.asciidoc

[id="{p}-uninstall"]
== Uninstall ECK

> Before you uninstall the operator, you must to uninstall the Elastic resources that it has created:

Reviewer: ```suggestion
Before uninstalling the operator, uninstall Elastic resources it has created:
```

> Before you uninstall the operator, you must to uninstall the Elastic resources that it has created:

Reviewer 2: ```suggestion
Before you uninstall the operator, you must remove all Elastic resources:
```

[source,shell]
----
kubectl delete elastic --all --all-namesapces
----

> This should delete all underlying Elasticsearch, Kibana or APM Server resources (pods, secrets, services, etc.).

Reviewer 2: ```suggestion
This deletes all underlying Elasticsearch, Kibana, and APM Server resources (pods, secrets, services, etc.).
```

> Then you can uninstall the operator itself:

Reviewer 3: Then, you can uninstall the operator:

[source,shell]
----
kubectl delete -f https://download.elastic.co/downloads/eck/0.9.0/all-in-one.yaml
----

> And terminate with the webhook configuration:

Reviewer 2: ```suggestion
And remove the webhook configuration:
```
